app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: int,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list[Car], income: float = 0) -> float:
+        for car in cars:
+            if car.clean_mark < self.clean_power:
```

Comment:
car clean_mark should be less or equal to wash station clean_power

Reply:
I'll change it, but don't you think it doesn't have any logic? like why do we need to clean a car if it's ```clean_mark == clean_power```, it's gonna do nothing to cleanliness of the car. And also because of that our income not gonna change cause you can see the screenshot lower with calculations where ```clean_mark == clean_power```

![NVIDIA_Share_iKUiap7D8l](https://github.com/mate-academy/py-car-wash-station/assets/13889031/7c57ab64-fdaa-42c8-8d61-f7fe9a49b85c)

Reply:
![NVIDIA_Share_50BM8RH4Gw](https://github.com/mate-academy/py-car-wash-station/assets/13889031/c52a3b68-fd89-4b68-84ed-a8ab4bb4cee4)
![NVIDIA_Share_qE7uZ8ftG7](https://github.com/mate-academy/py-car-wash-station/assets/13889031/06406a93-ffd7-4a88-aaa0-a8bd9a2ff61f)
![NVIDIA_Share_FsumFni5oy](https://github.com/mate-academy/py-car-wash-station/assets/13889031/3e20e067-16a2-440b-8260-2ea575eb5d76)